### DISCUSSION TITLE
Support Nextjs

### DISCUSSION THREAD

**Discussion by nirtamir2** [sandulat/routes-gen#30] [Ideas]
I notice https://github.com/tatethurston/nextjs-routes - which implements the pages+apps directory really good
We can create a plugin to routes-gen based on the code.

**Comment by sandulat** [comment]
@nirtamir2 This would be awesome!

**Comment by sandulat** [comment]
@nirtamir2 perhaps a NextJS driver would not make sense considering this is coming soon officially: https://twitter.com/shuding_/status/1620137501192253440

**Reply by nirtamir2** [reply]
It will be interesting to see what they come up with 👀
